### PR TITLE
[DAT-495] - Change input type for the "Industry" field to use "Select"

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Changed the "Industry" field to use "Select" instead of "Text"

**Before:**

image

**After:**

image

Currently the values for the Industries is obtained from ```https://cdn.fromdoppler.com/static-data/industries-es.json``` or ```industries-en.json```

**Task:** [DAT-495](https://makingsense.atlassian.net/jira/software/projects/DAT/boards/62?selectedIssue=DAT-495)

### CODE REVIEW

File: src/components/ChangePlan/Checkout/Checkout.test.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import IntlProvider from '../../../i18n/DopplerIntlProvider.double-with-ids-as-v
 import { AppServicesProvider } from '../../../services/pure-di';
 import { BrowserRouter } from 'react-router-dom';
 import Checkout from './Checkout';
+import { fakeIndustries } from '../../../services/static-data-client.double';
 
 describe('Checkout component', () => {
   afterEach(cleanup);
```
Comment (jcamposmk) [inline src/components/ChangePlan/Checkout/Checkout.test.js:12]: jest automatically does the cleanup, so you don't need to do it manually

File: src/components/ChangePlan/Checkout/Checkout.test.js [hunk 2]
```diff
@@ -34,6 +35,11 @@ describe('Checkout component', () => {
         return { success: true };
       },
     },
+    staticDataClient: {
+      getIndustriesData: async (language) => {
```
Comment (jcamposmk) [inline src/components/ChangePlan/Checkout/Checkout.test.js:39]:
```suggestion 
getIndustriesData: async (language) => ({ success: true, value: fakeIndustries }),
```

File: src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.js [hunk 3]
```diff
@@ -28,26 +28,45 @@ const fieldNames = {
...
       const fetchData = async () => {
         const contactInformationResult = await dopplerUserApiClient.getContactInformationData();
+        const industries = await getIndustries(intl.locale);
 
         setState({
           contactInformation: contactInformationResult.success
             ? contactInformationResult.value
             : null,
           success: contactInformationResult.success,
+          industries: industries,
```
Comment (jcamposmk) [inline src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.js:64]:
```suggestion
industries,
```

File: src/components/ChangePlan/Checkout/Checkout.test.js (hunk 1 above)
Comment (jhoffman-ms) [reply src/components/ChangePlan/Checkout/Checkout.test.js]: Done

File: src/components/ChangePlan/Checkout/Checkout.test.js (hunk 2 above)
Comment (jhoffman-ms) [reply src/components/ChangePlan/Checkout/Checkout.test.js]: Done

File: src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.js (hunk 3 above)
Comment (jhoffman-ms) [reply src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.js]: Done

**Comment by jcamposmk** [comment]
@kapoc3  The class that you created as a BaseClient, can it be applied in this PR?

File: .env.qa [hunk 4]
```diff
@@ -15,4 +15,5 @@ REACT_APP_DOPPLER_BILLING_API_URL=https://apisqa.fromdoppler.net/billing-api
 REACT_APP_MANUAL_STATUS_ENABLED=true
 REACT_APP_MANUAL_STATUS_FILE_URL = https://cdn.fromdoppler.com/webapp-status.json
 REACT_APP_DOPPLER_CONTACT_POLICY_URL=https://apisqa.fromdoppler.net/contact-policies
-REACT_APP_DOPPLER_BILLING_API_URL=https://apisqa.fromdoppler.net/doppler-users-api
\ No newline at end of file
+REACT_APP_DOPPLER_BILLING_API_URL=https://apisqa.fromdoppler.net/doppler-users-api
+REACT_APP_STATIC_DATA_BASE_URL = https://cdn.fromdoppler.com/static-data
```
Comment (cbernat) [inline .env.qa:19]: Why would you add this url as an environment variable? This is not changing now.

Comment (jhoffman-ms) [reply .env.qa:19]: Do you say that is enough add this variable only in ```.env```?

File: src/services/pure-di.tsx [hunk 5]
```diff
@@ -117,6 +118,7 @@ export class AppCompositionRoot implements AppServices {
       appStatusOverrideFileUrl: process.env.REACT_APP_MANUAL_STATUS_FILE_URL as string,
       dopplerContactPolicyApiUrl: process.env.REACT_APP_DOPPLER_CONTACT_POLICY_URL as string,
       dopplerUsersApiUrl: process.env.REACT_APP_DOPPLER_USERS_API_URL as string,
+      staticDataBaseUrl: process.env.REACT_APP_STATIC_DATA_BASE_URL as string,
```
Comment (cbernat) [inline src/services/pure-di.tsx:121]:
```suggestion
      staticDataBaseUrl: 'https://cdn.fromdoppler.com/static-data',
```

File: .env.qa (hunk 4 above)
Comment (jhoffman-ms) [reply .env.qa]: The  below review answered my question. I removed this url from all environment variables.

File: src/services/pure-di.tsx (hunk 5 above)
Comment (jhoffman-ms) [reply src/services/pure-di.tsx]: Done

**Comment by jhoffman-ms** [comment]
All reviews were fixed. Let me know if you have another comments.

**Comment by jcamposmk** [comment]
:+1:

**Pull request merged**